contracts/cw4-group/helpers.ts

```diff
@@ -0,0 +1,267 @@
+import axios from  "axios";
+import fs from "fs";
+import { SigningCosmWasmClient } from "@cosmjs/cosmwasm-stargate";
+import { GasPrice, calculateFee, StdFee } from "@cosmjs/stargate";
+import { DirectSecp256k1HdWallet, makeCosmoshubPath } from "@cosmjs/proto-signing";
+import { Slip10RawIndex } from "@cosmjs/crypto";
+import { toUtf8, toBase64 } from "@cosmjs/encoding";
+import path from "path";
+
+/*
+ * This is a set of helpers meant for use with @cosmjs/cli
+ * With these you can easily use the cw20 contract without worrying about forming messages and parsing queries.
+ *
+ * Usage: npx @cosmjs/cli@^0.26 --init https://raw.githubusercontent.com/CosmWasm/cw-plus/master/contracts/cw4-group/helpers.ts
+ *
+ * Create a client:
+ *   const [addr, client] = await useOptions(pebblenetOptions).setup('password');
+ *
+ * Get the mnemonic:
+ *   await useOptions(pebblenetOptions).recoverMnemonic(password);
+ *
+ * Create contract:
+ *   const contract = CW4Group(client, pebblenetOptions.fees);
+ *
+ * Upload contract:
+ *   const codeId = await contract.upload(addr);
+ *
+ * Instantiate contract example:
+ *   const initMsg = {
+ *     admin: addr,
+ *     members: [
+ *       {
+ *          addr: "wasm1hkxhcvw6sfyu6ztkce3dlz5nnk8kwjmcd7ettt",
+ *          weight: 10,
+ *       },
+ *       {
+ *          addr: "wasm1z6ms6cejaj8jz8zwkntx9ua0klhtptvz8elaxp",
+ *          weight: 15,
+ *       },
+ *     ]
+ *   };
+ *   const instance = await contract.instantiate(addr, codeId, initMsg, 'WORKFORCE1');
+ *
+ * If you want to use this code inside an app, you will need several imports from https://github.com/CosmWasm/cosmjs
+*/
+
+interface Options {
+  readonly httpUrl: string
+  readonly networkId: string
+  readonly feeToken: string
+  readonly bech32prefix: string
+  readonly hdPath: readonly Slip10RawIndex[]
+  readonly faucetUrl?: string
+  readonly defaultKeyFile: string,
+  readonly fees: {
+    upload: StdFee,
+    init: StdFee,
+    exec: StdFee
+  }
+}
+
+const pebblenetGasPrice = GasPrice.fromString("0.01upebble");
```

Comment:
pebble net config is repeated in each helper, right?

Reply:
Yes it is, but we are going to move this under a `base_helper.ts` soon https://github.com/InterWasm/DAO/issues/26